b/binaryextensions/binaryextensions_ubi_8.3.sh

```diff
@@ -0,0 +1,88 @@
+# ----------------------------------------------------------------------------
+#
+# Package               : binaryextensions
+# Version               : 2.2.0
+# Source repo           : https://github.com/bevry/binaryextensions
+# Tested on             : UBI 8.3
+# Script License        : Apache License, Version 2 or later
+# Passing Arguments     : Passing Arguments: 1.Version of package,
```

Comment:
Let's avoid changing the script header ..

Reply:
corrected the headers,pls check